CONCEPTS.md

```diff
@@ -0,0 +1,146 @@
+# Swagger Assert4j Core Concepts
+
+## JSON Recipes
+
+JSON recipes are the internal representation used by swagger-assert4j to describe API tests. A very simple example:
```

Comment:
Technically it's an "external representation". Should we just mention that you can also write them yourself as an Assert4J user?